parsers/ECO2MIX.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/env python3
+
+import json
+import math
+import os
+from datetime import datetime, timedelta
+from logging import Logger, getLogger
+
+import arrow
+import numpy as np
+import pandas as pd
+import requests
+from bs4 import BeautifulSoup
+
+from electricitymap.contrib.config import EXCHANGES_CONFIG, ZONES_CONFIG
+from parsers.lib.config import refetch_frequency
+from parsers.lib.exceptions import ParserException
+
+from .lib.validation import validate, validate_production_diffs
+
+# From RTE domain to EM domain
+MAP_ZONES = {
+    "France": "FR",
+    **{
+        subzone.replace("FR-", ""): subzone
+        for subzone in ZONES_CONFIG["FR"].get("subZoneNames", [])
```

Comment:
This creates some type errors as the Literal "FR" can't be assigned to the ZoneKey type but should not affect the functionality.